src/coredata.cs

```diff
@@ -627,6 +691,9 @@ public interface NSManagedObject {
 		[Export ("awakeFromInsert")]
 		void AwakeFromInsert ();
 
+		[Export ("awakeFromSnapshotEvents:")]
```

Comment:
It was added in iOs 3, did not add the availability attribute since it is not needed.